### ISSUE TITLE
Gradient clipping: why not global norm ?

### ISSUE THREAD

**Issue by sheetalgiri** [lucidrains/enformer-tensorflow-sonnet-training-script#1]
In the paper they say "We clipped gradients to a maximum global norm of 0.2."
In 
https://github.com/lucidrains/enformer-tensorflow-sonnet-training-script/blob/6de9af047ecc1d8158afb8f12d128c6d504c5511/train.py#L1060 you choose to do simple clip_by_norm with a value of 1.0 
I just wanted to ask the reasoning behind this choice, why did you not use tf.clip_by_global_norm instead ? 

Just wanted to know if it's something you already tried and chose against as I struggle to train the model properly myself(using enformer_pytorch)
